frontend/src/__tests__/cypress/cypress/pages/appLauncher.ts

```diff
@@ -0,0 +1,34 @@
+import { Contextual } from './components/Contextual';
+
+class ApplicationLauncherMenuGroup extends Contextual<HTMLElement> {
+  shouldHaveApplicationLauncherItem(name: string) {
+    this.find().findAllByTestId('application-launcher-item').contains(name).should('exist');
+    return this;
+  }
+}
+
+class ApplicationLauncher {
+  visit() {
+    cy.visit('/explore');
+    this.wait();
+  }
```

Comment:
Move this to function to `AppChrome` object and make it visit the root of the app.

```suggestion
  visit() {
    cy.visit('/');
    this.wait();
  }
```